pravega-client-examples/src/main/java/io/pravega/example/secure/SecureReader.java

```diff
@@ -0,0 +1,186 @@
+         *      "cert.pem" as the CA certificate, which is also provided on the server-side. If the cluster uses a
+         *      different CA (which it should), use that CA's certificate as the truststore instead.
+         *
+         *    - Also, the client config below disables host name verification. If the cluster's server certificates
+         *      have DNS names / IP addresses of the servers specified in them, you may turn this on. In a production
+         *      deployment, it is recommended to keep this on.
+         *
+         * Note about setting the client config for auth:
+         *    - The client config below is configured with an object of DefaultCredentials class. The user name
+         *      and password arguments passed to the object represent the credentials used for authentication
+         *      and authorization. The assumption we are making here is that the username is valid on the server,
+         *      the password is correct and the username has all the permissions necessary for performing the
+         *      subsequent operations.
+         */
+        ClientConfig clientConfig = ClientConfig.builder()
+                .controllerURI(this.controllerURI) // "tls://localhost:9090"
+
+                // TLS-related client-side configuration
+                .trustStore(this.truststorePath)
+                .validateHostName(this.validateHostName)
+
+                // Auth-related client-side configuration
+                .credentials(new DefaultCredentials(this.password, this.username))
+                .build();
+
+        System.out.println("Done creating a client config.");
+
+        // Everything below depicts the usual flow of reading events. All client-side security configuration is
+        // done through the ClientConfig object as shown above.
+
+        ClientFactory clientFactory = null;
+        ReaderGroupManager readerGroupManager = null;
+        EventStreamReader<String> reader = null;
+        try {
+            ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
+                    .stream(Stream.of(this.scope, this.stream))
+                    .disableAutomaticCheckpoints()
+                    .build();
+            System.out.println("Done creating a reader group config with specified scope: [" +
+                    this.scope +"] and stream name: [" + this.stream + "].");
+
+            String readerGroupName = UUID.randomUUID().toString().replace("-", "");
+            readerGroupManager = ReaderGroupManager.withScope(this.scope, clientConfig);
+            readerGroupManager.createReaderGroup(readerGroupName, readerGroupConfig);
+            System.out.println("Done creating a reader group with specified name  and config.");
+
+            clientFactory = ClientFactory.withScope(this.scope, clientConfig);
+            System.out.println("Done creating a client factory with the specified scope and client config.");
```

Comment:
Could you please replace all system out with logger?

Reply:
In a client example, I intentionally avoided loggers, to keep things as simple as possible. When copying the sample code for use in their own applications, there would be one less dependency and step to take care of. Besides, when running the program through CLI, this also results in just pertinent/minimum output.